src/main/scala/org/broadinstitute/dsde/vault/services/uBAM/DescribeServiceHandler.scala

```diff
@@ -14,7 +14,7 @@ import uBAMJsonProtocol._
 
 object DescribeServiceHandler {
   case class DescribeMessage(dmId: String)
-
+  case class DescribeMessageList()
```

Comment:
I would prefer the name `DescribeListMessage`, and it should also be `case object DescribeLikeMessage` , like we did for `DMResolveUBamList`